wp-includes/js/service-worker-navigation-routing.js

```diff
@@ -3,9 +3,9 @@ ERROR_OFFLINE_URL, ERROR_500_URL, SHOULD_STREAM_RESPONSE, STREAM_HEADER_FRAGMENT
 ERROR_OFFLINE_BODY_FRAGMENT_URL, STREAM_HEADER_FRAGMENT_QUERY_VAR, NAVIGATION_BLACKLIST_PATTERNS, ERROR_MESSAGES */
 
 {
-	const isStreamingResponses = SHOULD_STREAM_RESPONSE && wp.serviceWorker.streams.isSupported();
-	const errorMessages = ERROR_MESSAGES;
-	const navigationRouteEntry = NAVIGATION_ROUTE_ENTRY;
+	var isStreamingResponses = SHOULD_STREAM_RESPONSE && wp.serviceWorker.streams.isSupported();
+	var errorMessages = ERROR_MESSAGES;
+	var navigationRouteEntry = NAVIGATION_ROUTE_ENTRY;
```

Comment:
I just remembered where this was fixed before. In the AMP plugin an IIFE was used instead of switching to `var`: https://github.com/ampproject/amp-wp/commit/60b3bca47ad26eb8d82abb78a035cda72a6c1b51

Would you be willing to make a similar change to this file and the other JS files in this directory?

Reply:
Also, please add a comment before the IIFE to explain why it is used:

```js
// IIFE is used for lexical scoping instead of just a braces block due to bug with const in Safari.
( () => {
// ...
} )();
```

Reply:
Done. Thanks for the link... IIFE does seem like a better approach. :heart: